Xamarin.Forms.Platform.WinRT/DatePickerRenderer.cs

```diff
@@ -90,13 +90,25 @@ void UpdateDate(DateTime date)
 		void UpdateMaximumDate()
 		{
 			DateTime maxdate = Element.MaximumDate;
-			Control.MaxYear = new DateTimeOffset(maxdate.Date);
+			Control.MaxYear = new DateTimeOffset(maxdate);
 		}
 
 		void UpdateMinimumDate()
 		{
 			DateTime mindate = Element.MinimumDate;
-			Control.MinYear = new DateTimeOffset(mindate);
+
+			try
+			{
+				Control.MinYear = new DateTimeOffset(mindate);
+			}
+			catch (ArgumentOutOfRangeException)
```

Comment:
Turns out a try/catch is really just the simplest fix because the exception should only be thrown in this one scenario.